### ISSUE TITLE
Cannot close editor tabs with middle click

### ISSUE THREAD

**Issue by DesiQuintans** [rstudio/rstudio#4379]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1303
    OS Version      : Win 7 64-bit
    R Version       : 3.5.2

### Describe the problem in detail

This is a regression from the current stable. In the stable release, middle-clicking a tab in the editor pane closes the tab. This does not work in the Preview release.

**Comment by jmcphers** [comment]
Fixed in https://github.com/rstudio/rstudio/pull/4548.

**Comment by DesiQuintans** [comment]
Hi, this does not seem to be fixed in the newest stable (1.2.1335). 

```
RStudio Edition : Desktop
RStudio Version : 1.2.1335
OS Version      : Win 10 64-bit
R Version       : 3.5.3
```

**Comment by kevinushey** [comment]
The fix unfortunately didn't make it to the v1.2 release -- it will become part of a later patch release for RStudio v1.2.

**Comment by ronblum** [comment]
@DesiQuintans Would you be willing to check if this is resolved in 1.2.1522-1, the [latest preview release](https://www.rstudio.com/products/rstudio/download/preview/)?

**Comment by DesiQuintans** [comment]
Fixed on 1.2.1522-1, thanks!

**Comment by ronblum** [comment]
@DesiQuintans Thank you for checking!
